src/Microsoft.Performance.SDK.Runtime.NetCoreApp/Plugins/PluginsLoader.cs

```diff
@@ -108,7 +116,7 @@ public IEnumerable<CustomDataSourceReference> LoadedCustomDataSources
         ///     <para/>
         ///     This method does not report an error if a different version of one of the custom data
         ///     sources being loaded has already been loaded by either the current or a previous call to <see cref="TryLoadPlugin(string)"/>
-        ///     or <see cref="TryLoadPlugins(IEnumerable{string}, out IList{string})"/>.
+        ///     or <see cref="TryLoadPlugins(IEnumerable{string}, out IDictionary{string,ErrorInfo})"/>.
         /// </remarks>
         public bool TryLoadPlugin(string directory)
```

Comment:
We should probably change this signature to take in as a parameter `out IList<ErrorInfo> errors` now that we can actually give helpful information

Reply:
This class will return one ErrorInfo per directory, and that ErrorInfo can have multiple additional errors in its Details property. I was trying to preserve the "this directory had an error" approach that the old signature implied.

Reply:
Ah, yep, my bad. I thought the other method was a mapping from string -> List<ErrorInfo>

In this case, maybe add an `out ErrorInfo` parameter that is something like `ErrorInfo.None` if there's no errors (specifying that if this method returns `true` the error info will be `None`)

Reply:
Sounds good, will do.